packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.types.ts

```diff
@@ -20,56 +20,81 @@ export enum YppAuthorizationErrorCode {
   CHANNEL_CRITERIA_UNMET_VIDEOS = 'CHANNEL_CRITERIA_UNMET_VIDEOS',
   CHANNEL_CRITERIA_UNMET_CREATION_DATE = 'CHANNEL_CRITERIA_UNMET_CREATION_DATE',
   YOUTUBE_QUOTA_LIMIT_EXCEEDED = 'YOUTUBE_QUOTA_LIMIT_EXCEEDED',
+  YOUTUBE_API_NOT_CONNECTED = 'YOUTUBE_API_NOT_CONNECTED',
+  QUERY_NODE_NOT_CONNECTED = 'QUERY_NODE_NOT_CONNECTED',
 }
 
 export type YppRequirementsErrorCode = Omit<
   YppAuthorizationErrorCode,
-  YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED | YppAuthorizationErrorCode.CHANNEL_NOT_FOUND
+  | YppAuthorizationErrorCode.CHANNEL_ALREADY_REGISTERED
+  | YppAuthorizationErrorCode.CHANNEL_NOT_FOUND
+  | YppAuthorizationErrorCode.YOUTUBE_QUOTA_LIMIT_EXCEEDED
+  | YppAuthorizationErrorCode.YOUTUBE_API_NOT_CONNECTED
+  | YppAuthorizationErrorCode.QUERY_NODE_NOT_CONNECTED
```

Comment:
Let's use `Pick` here instead of `Omit`, it's much easier to read what's actually inside this type